ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -363,6 +364,7 @@ measurement_descriptor:
   operating: "State when a decvice active"
   hue: "degree on color wheel from 0 to 360°"
   orientation: "Description of cardinal direction"
+  vibration: "Periodic back-and-forth motion."
```

Comment:
Please use `linearacceleration`  see example of existing vibration sensor here: `HVAC/SENSOR_LV3A`